app/views/statistics/index.html.erb

```diff
@@ -1,40 +1,116 @@
+        <h3>Group statistics</h3>
+          <p>Use the navbar on the left to view usage and cost statistics for all of the groups you are a part of. </p>
+      </div>
+      <div class="tab-pane fade" id="personal" role="tabpanel" aria-labelledby="personal-tab">
+        <h3>Personal statistics</h3>
+        <p>View your personal statistics below.</p>
+        <hr>
+        <div class="row stat-row">
+          <div class="col col-8 stat-col-data">
+            <%= pie_chart current_user.subscriptions.group(:subscription_name).sum(:cost_per_month), 
+            colors: ['#69193E', '#973756', '#572F41', '#BBAEAE', '#6F8A79', '#BDDDC9'],
+            legend: "left"
+            %>  
+          </div>
+          <div class="col col-4 stat-col-info">
+            <h4>Cost of Your Monthly Subscriptions</h4>
+            <p>Your total monthly cost of serivces on our website is <b>$<%= current_user.total_cost %></b>. 
+            The breakdown of where you spend your money can be seen on the pie chart for this section. 
+            Hover over the pie sections or view the table below to see where you spend your money or view the table below for a more indepth analysis.</p>
+          </div>
 
-      <%= pie_chart Subscription.group(:subscription_name).count, donut: true, legend: "left", colors: ['#69193E', '#973756', '#572F41', '#BBAEAE', '#6F8A79', '#BDDDC9'] %> 
-      <hr>
-      <h4>Total subscription costs</h4>
+        </div>
+        
+        <div class="row stat-row">
+          <div class="col col-4 stat-col-info">
+            <h4>Total Cost of Your Subscriptions</h4>
+            <p>Your total all-time cost of serivces tracked on our website is <b>$<%= current_user.total_cost_overall %></b>.
+            The table in this section shows the cost you pay per month, the total amount of money you've spent on a subscription service since adding it to Gatekeeper, 
+            the service's website, and the day you added the subscription to our website.</p>
+          </div>
+          <div class="col col-8 stat-col-data">
+            <table class="table">
+              <thead class="stat-table">
+                <tr>
+                  <th scope="col">Cost / mo</th>
+                  <th scope="col">Total Cost</th>
+                  <th scope="col">Subscription Name</th>
+                  <th scope="col">Website</th>
+                  <th scope="col">Created On</th>
+                </tr>
+              </thead>
+              <tbody>
+              <% current_user.subscriptions.order(:cost_per_month).each do |sub| %>
+                  <tr>
+                    <td>$<%= sub.cost_per_month %></td>
+                    <td>$<%= ((((Time.now - sub.created_at) / 60 / 60 / 24).round / 30 + 1) * sub.cost_per_month) %></td>
+                    <td><%= sub.subscription_name %></td>
+                    <td><a href="https://<%= sub.url %>" target=_blank><%= sub.url %></a></td>

```

Comment:
use regex to fix this. that way if the user enters a url that has the https:// it doesn't break. i'm actually including a fix for this if you just want to remove the additional string
```suggestion
                    <td><a href="<%= sub.url %>" target=_blank><%= sub.url %></a></td>

```

Reply:
Gotcha, that makes sense! Just changed it